Desktop/html/css/printing.css

```diff
@@ -1,24 +1,30 @@
 @media only print
 {
-    #results {  display: block;  }
+    #results    {  display: block;  }
 
     .jasp-toolbar h1, .jasp-toolbar h2, .jasp-toolbar h3, .jasp-toolbar h4, .jasp-toolbar h5, .jasp-toolbar h6, .jasp-toolbar div , .jasp-toolbar span
-	                                { display:				block !important;	}
-	.toolbar-button					{ display:				none  !important;	visibility: hidden; }
-	.jasp-menu						{ display:				none  !important;	visibility: hidden; }
-	.jasp-menu-selected				{ display:				none  !important;	visibility: hidden; }
-	.jaspReportTop					{ display:				none  !important;	visibility: hidden; }
-	body							{ float:				none  !important;	}
+	                                    { display:	block !important;  }
+	.toolbar-button			    		{ display:	none  !important;  visibility: hidden; }
+	.jasp-menu			    			{ display:	none  !important;  visibility: hidden; }
+	.jasp-menu-selected		    		{ display:	none  !important;  visibility: hidden; }
+	.jaspReportTop			    		{ display:	none  !important;  visibility: hidden; }
+	body				   				{ float:	none  !important;  }
+
+    .jasp-toolbar			    		{ page-break-after:	avoid;	}
+	.jasp-analysis			    		{ page-break-after:	always; }
+	.jasp-collection		    		{ page-break-inside:	avoid;	}
+	.hidden-collection		    		{ page-break-inside:	auto;	}
+	.object-body			    		{ page-break-inside:	auto;	}
+	.object-body .hidden-collection	    { page-break-inside:	auto;	}
+	.jasp-image			    			{ page-break-inside:	avoid;	}
+	.jasp-notes			    			{ page-break-inside:	avoid;	}
+
+	.jasp-table-primitive		   		{ page-break-inside:	avoid;	}
+    .jasp-table table               	{ table-layout:fixed !important;  width: 90em !important; }
+    .jasp-table table td            	{ white-space: pre-wrap !important; word-wrap: break-all !important; }
+
+	.jasp-rsyntax-container 			{ max-width: 90em !important; }
```

Comment:
What is this 90em coming from?

Reply:
Haha, good point! This is what I calculated based on the A4 paper size - 1cm and the font size. But I rethought it and realized that this is unreasonable and should be set directly to the paper size.

Any suggestions?